### ISSUE TITLE
Error on installating the app in the site

### ISSUE THREAD

**Issue by Tharun1718** [aakvatech/transport#18]
I have erpnext app installed already in my app and frappe is in there by default and while trying to install the trans_ms app.. I am getting this error..Please can you throw some light on this..Thanks
![image](https://github.com/aakvatech/transport/assets/108926726/82fb9f44-8699-4cac-988b-68cbd03a0d1a)

**Comment by av-dev2** [comment]
@Tharun1718  Normally the error above occurs when you install the app twice or more

make sure trans_ms is full uninstalled from the site and then install it again

**Comment by Tharun1718** [comment]
Thanks. I tried a fresh installation and I am getting the below error while installing the app in the site. I have used your branch version 14

![image](https://github.com/aakvatech/transport/assets/108926726/29a0b648-d8f7-4bbb-8d5f-e849cb536452)

**Comment by Tharun1718** [comment]
@av-dev2 can you please help on the above query

**Comment by lovechigwanda** [comment]
I am also facing the same error even after using branch version 14. Did you manage to get a response or workaround?

**Comment by Tharun1718** [comment]
@lovechigwanda nope, I didn't get any response. Please let me know if you got any in the future. Thanks

**Comment by guinanlin** [comment]
> Thanks. I tried a fresh installation and I am getting the below error while installing the app in the site. I have used your branch version 14
> 
> ![image](https://user-images.githubusercontent.com/108926726/242165931-29a0b648-d8f7-4bbb-8d5f-e849cb536452.png)

not install telegram but https://docs.python-telegram-bot.org/en/v20.5/index.html
